packages/api/scripts/test-bidirectional-edges.js

```
@@ -1,181 +1,181 @@
+  try {
+    // Test database connection
+    console.log('1️⃣ Testing database connection...');
+    const isConnected = await database.testConnection();
+    if (!isConnected) {
+      throw new Error('Database connection failed');
+    }
+    console.log('✅ Database connected\n');
+
+    // Test case 1: Ground floor elevator node (should connect up and down)
+    console.log('2️⃣ Testing Ground Floor Elevator (Node 10)...');
+    const groundElevatorEdges = await testNodeEdges(10, 'Ground Floor Elevator');
```

ESLint

[error] 'groundElevatorEdges' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)